api/src/main/resources/workbench-api.yaml

```diff
@@ -5392,11 +5407,11 @@ definitions:
 
 '
         type: string
-  RecentResourceResponse:
+  WorkspaceResourceResponse:
```

Comment:
I didn't look closely at `RecentResource` before, but I don't think it's a pattern we want to extend without careful consideration.

Instead of having an array of this type, I would've thought you'd want (instead of an overall array) a single object with an array of Cohort Reviews, Concept Sets, etc. You can certainly array-ify them on the front end if you need to.

If I'm reading this right, you've basically got a C language `union`-style structure, where at most one of each category is populated. That's not a great abstraction in a language where you don't have private fields. But I guess there's no polymorphism in Swagger.

Reply:
Yeah, its not ideal. It would be good if we could completely change this, but that would be a breaking api change in several places. Which I am okay doing, but I think for now I'd rather roll it out using the existing structure. 

I could roll a new structure that uses this: https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/ ? I don't know if that is usable in swagger codegen, but I can try. (Just call it `WorkspaceResourceV2` or something.)

Reply:
Discussed offline. Looks like polymorphism isn't supported in OpenApi 2, and moving to an object-aware model would be a big refactor on the front and backend, so that will spin off into its own ticket. https://precisionmedicineinitiative.atlassian.net/browse/RW-4740

Reply:
Filed https://precisionmedicineinitiative.atlassian.net/browse/RW-4740 to track this. Decided to wait until after OpenApi 3.0 and discussion on what a workspace is.